lib/device_map.py

```diff
@@ -261,11 +261,9 @@
            'mach': 'exynos'}
 
 snow = {'device_type': 'snow',
-        'templates': ['generic-arm-dtb-kernel-ci-boot-template.json',
-                      'generic-arm-dtb-kernel-ci-ltp-mm-template.json',
-                      'generic-arm-dtb-kernel-ci-ltp-syscalls-template.json',
-                      'generic-arm-dtb-kernel-ci-kselftest-template.json',
-                      'generic-arm-dtb-kernel-ci-hackbench-template.json'],
+        'templates': ['generic-uboot-tftp-ramdisk-template.jinja2',
+                      'generic-uboot-tftp-ramdisk-kselftest-template.jinja2',
+                      'generic-uboot-tftp-nfs-template.jinja2'],
```

Comment:
Any reason you added NFS boots here? We weren't doing them previously. Do you know they work?

Reply:
Well I did the same as on other boards such as the `r8a7795_salvator_x`, as in this change:
https://github.com/kernelci/lava-ci-staging/commit/6d4195e2cb7607d9bb03a0a83e9e9451d4a1c820#diff-61ccdff13aae0ec1982fb94d4c5170e8L698

If we've now decided to not run with NFS root then we should probably align all the boards.  Otherwise, if it works then why not test it?

Reply:
I added those because they either were already booting NFS in v1, or I knew they could boot NFS in v2. So same question as before, do you know these work? If so, it's fine to add it

Reply:
Well I believe these boards can boot over NFS, and we can test that on staging before deploying it in production.